app/views/components/docs/content-metadata.yml

```diff
@@ -31,16 +33,31 @@ examples:
       link_to_history: true
   two_publishers:
     data:
-      from:
-        - <a href='/government/organisations/department-for-education'>Department for Education</a>
-        - <a href='/government/organisations/education-funding-agency'>Education Funding Agency</a>
       published: 31 July 2017
       last_updated: 20 September 2017
+      other:
+        From:
+          - <a href='/government/organisations/department-for-education'>Department for Education</a>
+          - <a href='/government/organisations/education-funding-agency'>Education Funding Agency</a>
   multiple_publishers_and_no_link_to_history:
     data:
-      from:
-        - <a href='/government/organisations/department-for-education'>Department for Education</a>
-        - <a href='/government/organisations/education-funding-agency'>Education Funding Agency</a>
-        - <a href='/government/organisations/department-for-work-pensions'>Department for Work and Pensions</a>
       published: 31 July 2017
       last_updated: 20 September 2017
+      other:
+        From:
+          - <a href='/government/organisations/department-for-education'>Department for Education</a>
+          - <a href='/government/organisations/education-funding-agency'>Education Funding Agency</a>
+          - <a href='/government/organisations/department-for-work-pensions'>Department for Work and Pensions</a>
+  nothing:
+    description: |
```

Comment:
Should this be an example or just a line in the intro to the component? I can see why you'd consider the fail state and test against it but specifically presenting an example seems counter intuitive to me.

Reply:
I think it's appropriate to keep to avoid the risk of future regressions.